inc/agent.class.php

```diff
@@ -139,12 +137,32 @@ function rawSearchOptions() {
             'field'         => 'port',
             'name'          => _n('Port', 'Ports', 1),
             'datatype'      => 'integer',
+         ], [
+            'id'            => '15',
+            'table'         => $this->getTable(),
+            'field'         => 'ip_binary',
+            'name'          => __('Ip'),
+            'datatype'      => 'specific',
+            'nosearch'      => true,
+            'massiveaction' => false,
          ]
       ];
 
       return $tab;
    }
 
+   static function getSpecificValueToDisplay($field, $values, array $options = []) {
+      if (!is_array($values)) {
+         $values = [$field => $values];
+      }
+      switch ($field) {
+         case 'ip_binary':
+         return inet_ntop($values[$field]);
```

Comment:
You should probably handle the case `inet_ntop()` returns `false` in case of error.